jekyll/_cci2/testing-orbs.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: classic-docs
+title: "Testing CircleCI Orbs"
+short-title: "Testing Orbs"
+description: "Starting point for Testing CircleCI Orbs"
+categories: [getting-started]
+order: 1
+---
+## Introduction
+
+CircleCI orbs are packages of configuration that you can use in your configurations to simplify your workflows and quickly and easily deploy application and orbs in your workflow. When creating orbs for your workflow, you will want to perform testing to ensure your orb meets your specific needs before deploying and publishing the orb.
+
+You can think of orb testing at 4 levels, in increasing levels of complexity and scope.
+
+* Schema Validation - this can be done with a single CLI command and checks if the orb is well-formed YAML and conforms to the orb schema.
+* Expansion Testing - this can be done by scripting the CircleCI CLI and tests whether the elements of the orb generate the configuration you intended when processing configuration containing those elements.
+* Runtime Testing - this requires setting up separate tests and running them inside a CircleCI build.
+* Integration Testing - this is likely only needed for fairly advanced orbs or orbs designed specifically as public, stable interfaces to 3rd-party services. Doing orb integration tests requires a custom build and your own external testing environments for whatever systems you are integration with.
+
+The sections below provide details on each of these techniques for each level of orb testing.
+
+### Schema Validation
+
+To test whether an orb is valid YAML and is well-formed according to the schema, use `circleci orb validate` with the CircleCI CLI.
+
+#### Example
+
+Given an orb with source at `./src/orb.yml` you can run `circleci orb validate ./src/orb.yml` to receive feedback on whether the orb is valid and will pass through config processing. If there is an error, you will receive the first schema validation error encountered. Alternatively, you can pass STDIN rather than a file path. 
+
+For example, equivalent to the previous example you can run `cat ./src/orb.yml | circleci orb validate.' 
```

Comment:
this straight tick should be a backtick to correct the formatting

Reply:
Fixed